selfdrive/car/honda/interface.py

```diff
@@ -33,12 +32,12 @@ def get_pid_accel_limits(CP, current_speed, cruise_speed):
       return CarControllerParams.NIDEC_ACCEL_MIN, interp(current_speed, ACCEL_MAX_BP, ACCEL_MAX_VALS)
 
   @staticmethod
-  def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
+  def _get_params(ret, candidate: CAR, fingerprint, car_fw, experimental_long, docs):
     ret.carName = "honda"
 
     CAN = CanBus(ret, fingerprint)
 
-    if candidate in HONDA_BOSCH:
+    if candidate.config.flags & HondaFlags.BOSCH:
```

Comment:
You can create/set these flags in a new PR to reduce the diff if you'd like

Reply:
yea, done